modules/ROOT/pages/apid-behavioral-headers.adoc

```diff
@@ -34,16 +30,51 @@ Examples:
 }
 ----
 
-MS2-Error-Level:: Indicates which range of error status codes the mocking service is restricted to using. You can specify one of the following three case-sensitive values:
-+
+== MS2-Example
+
+The MS2-Example header enables you to choose a specific response example if your endpoint defines more than one example for the same status code.
```

Comment:
"for the same status code" -> "for a single status code"

Reply:
Changed.!